doc(release) Release notes collect 23.10.16 gorgone 23.10.13 (#4496)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-23.10/releases/centreon-os.mdx

## Centreon Collect

### 23.10.16

> Release date: `July 15, 2025`

Reviewer: ```suggestion
Release date: `July 16, 2025`
```

<details>
  <summary>Bug fixes</summary>

> - [Resource status] Fixed issues on tags (filters on HG, HC, SG, SC).

Reviewer: ```suggestion
- [Resource status] Fixed issues on tags (filters on host groups, host categories, service groups, service categories).
```

## Centreon Gorgone

### 23.10.13

> Release date: `July 15, 2025`

Reviewer: ```suggestion
Release date: `July 16, 2025`
```

<details>
  <summary>Bug fixes</summary>

> - Fixed a bug on alma9 where plugin where not installed because yum don't exist anymore.

Reviewer: ```suggestion
- Fixed an issue on Alma 9 where plugins were not installed because yum did not exist anymore.
```
